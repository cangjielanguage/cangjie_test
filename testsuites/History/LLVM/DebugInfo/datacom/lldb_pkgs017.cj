// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_pkgs017_expected.txt ../packages/pkgs007/src ../packages/pkgs007/cfg-ast
// EXEC: cjc -c -g src/module007/src/pkgs/pkgs.cj -module-name module007
// EXEC: cjc -g  src/pkgs007.cj -import-config cfg-ast pkgs.o
// EXEC: python3 DatacomTest.py lldb_pkgs017_expected.txt

