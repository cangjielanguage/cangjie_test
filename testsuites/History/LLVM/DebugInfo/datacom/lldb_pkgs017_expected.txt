lldb,lldb pkgs007,pkgs007
b,b 12,pkgs007.cj:12
r,r,-> 12
n,n,-> 13
cj_plocals,cj_plocals,a = 1
cj_plocals,cj_plocals,b = 2